src/test/java/org/jenkinsci/plugins/workflow/support/steps/WorkspaceStepTest.java

```diff
@@ -56,7 +56,7 @@ public class WorkspaceStepTest {
     @Test public void customWorkspaceConcurrency() throws Exception {
         // Currently limited to WorkspaceList.allocate:
         WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
-        // Use master as it has 2 executors by default, whereas createSlave hardcodes 1, and I do not want to bother creating a slave by hand:
+        // Use controller as it has 2 executors by default, whereas createSlave hardcodes 1, and I do not want to bother creating a agent by hand:
```

Comment:
@MarkEWaite I wonder if this one is not the referring to the "master node" and not the "master instance", and should be called "built-in node". "controller" looks good to and is maybe more easy to understand for more plugin developpers.

```suggestion
        // Use the controller as it has 2 executors by default, whereas createSlave hardcodes 1, and I do not want to bother creating an agent by hand:
```